rl4co/models/zoo/mdam/decoder.py

```diff
@@ -85,8 +86,9 @@ def __init__(
             env_name, {"embedding_dim": embedding_dim}
         )
 
-        self.logit_attention = [
-            LogitAttention(
+        # MHA with Pointer mechanism (https://arxiv.org/abs/1506.03134)
+        self.pointer = [
```

Comment:
seems like the pointers in self.pointer are never used. Probably, they should replace the "_one_to_many_logits" function?

Reply:
(As discussed, left as TODO for the future for the time being!)

Reply:
Yes I think this kind of redundancy may come from the transferring from the original [MDAM implementation](https://github.com/liangxinedu/MDAM/blob/19b0bf813fb2dbec2fcde9e22eb50e04675400cd/nets/model_search.py#L51). Let me check it then.